misc/src/main/scala/com/danieltrinh/scalariform/gui/FormatterFrame.scala

```diff
@@ -294,6 +318,11 @@ class FormatterFrame extends JFrame with SpecificFormatter {
             preferences = preferences.setPreference(prefType.cast(preference), widget.asInstanceOf[JCheckBox].isSelected)
           case prefType @ IntegerPreference(min, max) ⇒
             preferences = preferences.setPreference(prefType.cast(preference), Integer.parseInt(widget.asInstanceOf[JSpinner].getValue.toString))
+          case prefType @ IntentPreference ⇒
+            val selected = widget.asInstanceOf[JPanel].getComponents.find(_.asInstanceOf[JRadioButton].isSelected).get.getName
```

Comment:
This line seems to fail when rendering the gui with "sbt> project misc; run;". I'm not familiar enough with swing to know the proper way of accessing the radio buttons, but doing a `.getComponents` here seems to return an empty array.